docs/backend/fields.md

```diff
@@ -103,6 +103,25 @@ See [`plone.namedfile`](https://pypi.org/project/plone.namedfile/) and [plone.fo
 | `NamedBlobFile` | `NamedBlobFile` | A binary uploaded file stored as a ZODB blob. Requires the `blobs` extra to `plone.namedfile`. Otherwise identical to `NamedFile`. | `IField` |
 | `NamedBlobImage` | `NamedBlobImage` | A binary uploaded image stored as a ZODB blob. Requires the `blobs` extra to `plone.namedfile`. Otherwise identical to `NamedImage`. | `IField` |
 
+#### NamedBlobImage
+
+The following example shows, how to create an Image object and attach the image file to it.
+
+```python
+img_obj = api.content.create(
+    container=ww_article,
+    type="Image",
+    id="test.jpg",
+)
+img_obj.image = NamedBlobImage(
+    data=img_file,
+    filename="test.jpg",
+)
```

Comment:
It's probably better to pass the image in the call to create(), so that the image_scales get indexed during the object creation. (Otherwise you would need a separate reindex after this)

Reply:
@MrTango would you please respond to @davisagli's suggestion?

Reply:
@davisagli thx for the hint, better now?